src/components/Tooltip/index.js

```diff
@@ -0,0 +1,62 @@
+import React, {Component} from 'react'
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+
+export default class Tooltip extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      active: false,
+    };
+    this.toggle = this.toggle.bind(this);
+  }
+
+  toggle(e) {
+    this.setState({ active: !this.state.active });
+  }
+
+  render() {
+    const { position, small, trigger } = this.props;
+    const { active } = this.state;
+    const parentClassName = cx({
+      'us-tooltip': true,
+      'us-tooltip--active': active,
+      'us-tooltip--top': position === 'top',
+      'us-tooltip--bottom': position === 'bottom',
+      'us-tooltip--right': position === 'right',
+      'us-tooltip--left': position === 'left',
```

Comment:
This is a stylistic thing, but for this position stuff, you could do something like

```js
  // ...
  [`us-tooltip--${position}`]: !!position
  // ...
```

And then enumerate the different position types in the `PropTypes` as

```js
  position: PropTypes.oneOf([ 'left', 'right', 'bottom', 'top' ])
```

To me this seems more concise/explicit of the API _(don't have to update the `cx` call to add more position variants as classes)_ but I understand that it's perhaps trying to be _"clever"_ and probably makes it harder to read 🙂

Reply:
I like that. I'll make those changes